lekko_client/clients/client.py

```diff
@@ -38,7 +38,7 @@ def get_string(self, namespace: str, key: str, context: Dict[str, Any]) -> str:
         ...
 
     @abstractmethod
-    def get_json(self, namespace: str, key: str, context: Dict[str, Any]) -> dict:
+    def get_json(self, namespace: str, key: str, context: Dict[str, Any]) -> dict[str, Any]:
```

Comment:
Doesn't have to be part of this set of changes, but we should unify the typing to use dict vs Dict and list vs List since we only support 3.10+ anyway.

Reply:
yeah, I can do a separate sweep to make it consistent